test/unit/data/standardFlairInfo.json

```diff
@@ -0,0 +1,54 @@
+    "cherishball":{"name":"cherishball","sub":"pokemontrades","trades":80,"shinyevents":65,"events":25,"eggs":0,"giveaways":0,"involvement":0},
+    "ovalcharm":{"name":"ovalcharm","sub":"pokemontrades","trades":90,"shinyevents":0,"events":0,"eggs":0,"giveaways":0,"involvement":0},
+    "shinycharm":{"name":"shinycharm","sub":"pokemontrades","trades":100,"shinyevents":140,"events":70,"eggs":0,"giveaways":0,"involvement":0},
+    "involvement":{"name":"involvement","sub":"pokemontrades","giveaways":5,"involvement":10,"trades":0,"eggs":0,"shinyevents":0,"events":0},
+    "lucky":{"name":"lucky","sub":"svexchange","eggs":1,"trades":0,"shinyevents":0,"events":0,"giveaways":0,"involvement":0},
+    "egg":{"name":"egg","sub":"svexchange","eggs":5,"trades":0,"shinyevents":0,"events":0,"giveaways":0,"involvement":0},
+    "eevee":{"name":"eevee","eggs":10,"sub":"svexchange","trades":0,"shinyevents":0,"events":0,"giveaways":0,"involvement":0},
+    "togepi":{"name":"togepi","eggs":20,"sub":"svexchange","trades":0,"shinyevents":0,"events":0,"giveaways":0,"involvement":0},
+    "torchic":{"name":"torchic","eggs":30,"sub":"svexchange","trades":0,"involvement":0,"giveaways":0},
+    "pichu":{"name":"pichu","sub":"svexchange","eggs":50,"trades":0,"involvement":0,"giveaways":0},
+    "manaphy":{"name":"manaphy","sub":"svexchange","eggs":75,"trades":0,"shinyevents":0,"events":0,"giveaways":0,"involvement":0},
+    "eggcup":{"name":"eggcup","sub":"svexchange","eggs":100,"trades":0,"shinyevents":0,"events":0,"giveaways":0,"involvement":0},
+    "cuteribbon":{"sub":"svexchange","name":"cuteribbon","eggs":0,"giveaways":30,"trades":0,"shinyevents":0,"events":0,"involvement":0},
+    "coolribbon":{"sub":"svexchange","name":"coolribbon","eggs":0,"giveaways":100,"trades":0,"shinyevents":0,"events":0,"involvement":0},
+    "beautyribbon":{"sub":"svexchange","name":"beautyribbon","eggs":0,"giveaways":200,"trades":0,"shinyevents":0,"events":0,"involvement":0},
+    "smartribbon":{"name":"smartribbon","sub":"svexchange","giveaways":400,"eggs":0,"trades":0,"involvement":0},
+    "toughribbon":{"name":"toughribbon","sub":"svexchange","giveaways":800,"eggs":0,"trades":0,"involvement":0}
+  },
+  "expectedFormat":{
+    "pokeball":"Poke Ball",
+    "premierball":"Premier Ball",
+    "greatball":"Great Ball",
+    "ultraball":"Ultra Ball",
+    "luxuryball":"Luxury Ball",
+    "masterball":"Master Ball",
+    "dreamball":"Dream Ball",
+    "cherishball":"Cherish Ball",
+    "ovalcharm":"Oval Charm",
+    "shinycharm":"Shiny Charm",
+    "involvement":"Involvement",
+    "lucky":"Lucky Egg",
+    "egg":"Egg",
+    "eevee":"Eevee Egg",
+    "togepi":"Togepi Egg",
+    "torchic":"Torchic Egg",
+    "pichu":"Pichu Egg",
+    "manaphy":"Manaphy Egg",
+    "eggcup":"Egg Cup",
+    "cuteribbon":"Cute Ribbon",
+    "coolribbon":"Cool Ribbon",
+    "beautyribbon":"Beauty Ribbon",
+    "smartribbon":"Smart Ribbon",
+    "toughribbon":"Tough Ribbon"
+  }
+}
```

Comment:
This reminds me: Do we really need a separate database collection for flairs? They change very infrequently -- it seems like it would be easier just to have them hard-coded into a json file, and have the formatted names explicitly stored with them rather than relying on parsing the regular names.